src/components/header/header.jsx

```diff
@@ -210,7 +210,7 @@ Header.Profile = () => {
         <div className="flex flex-row h-[24px] gap-2.5 md:gap-1 md:pl-4 md:flex lg:flex sm:hidden xs:hidden">
           <Button
             className={"border-b-2"}
-            size="sm"
+            size="base"
```

Comment:
Login and Signup looks good, however the text should look like the other header links but other than that the underline is correct